scripts/boot.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+# mark as executable
+chmod a+x scripts/setup.sh
+
+# echo now
+echo "Starting provision service that will run every few hours/minutes to provision apps"
+
+# continues while loop
+while :
+do
+
+	# debug
+	echo "kill all previous runs that are still active of setup"
+
+	# kill the running instance
+	pkill -9 -f setup.sh
```

Comment:
Doesn't `-9` create the potential for zombie processes?
Perhaps `-15` to give the process some time to clean up after itself and shut down as best as possible?